app/Console/Commands/MoveAvatars.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Console\ConfirmableTrait;
+
+class MoveAvatars extends Command
```

Comment:
This is something that should be done only once, right? Maybe we could indicate it in the description.

Reply:
As you know, the location of an avatar is stored in ``Contact -> avatar_location`` property. The ``MoveAvatars`` command aims to move avatars (contacts with ``has_avatar`` to true) from current location to the - possibly different - default driver, as ``config('filesystems.default')``. This way, you can run the commands multiple times: if the current location of the avatar is the default one, it's not touched.

Reply:
But will we have to run the commands multiple times?

Reply:
We will have to run the command one time for this migration, but it could be run multiple times, every time you update the default filesystem store.